### ISSUE TITLE
Red square covering stone whenever a stone is placed

### ISSUE THREAD

**Issue by limsherwin** [upsided/Upsided-Sabaki-Themes#1]
Hello,

Happy Stones and BadukTV themes have red square covering the recently placed stone.
Hikaru theme does not have this issue.

Currently using Sabaki Version 0.52.2 (0.52.2)

Cheers,
Sherwin

<img width="807" alt="Screen Shot 2022-11-16 at 9 53 59 AM" src="https://user-images.githubusercontent.com/941004/202064306-594d3a64-e273-43f5-8265-ab7dbfe31fac.png">
<img width="807" alt="Screen Shot 2022-11-16 at 9 54 22 AM" src="https://user-images.githubusercontent.com/941004/202064319-b969586d-66ca-4d09-9e74-6fdb758ff055.png">

**Comment by zengxiaofei** [comment]
Very nice theme 👍, but I have the same problem. 😢

**Comment by zengxiaofei** [comment]
I guess the problem emerged due to the way that Sabaki parses CSS files changed in the later version. I don't know how to correct it perfectly because I'm not familiar with web design and CSS. I just directly replaced the color of the "latest move indicator", `#b90202`, with `transparent` in the asar file. It works well for me, although the indicator color becomes black and white again.

A screenshot of my modified version of BadukTV theme:
![modified](https://user-images.githubusercontent.com/22426230/223144123-6fb69e9a-a85b-4fc0-9e92-eb64e6698a90.png)

The modified asar file of BadukTV theme:
[baduktv.zip](https://github.com/upsided/Upsided-Sabaki-Themes/files/10899433/baduktv.zip)

**Comment by apetresc** [comment]
Indeed, the problem seems to have begun with the Shudan 1.7.0 upgrade which changed the stone graphics to SVGs instead of images. Whatever CSS rules are there seem to conflict with the SVGs for the `.shudan-marker`s your styles use.

**Comment by apetresc** [comment]
Based on some quick prototyping in the developer panel, it seems like you just need to replace

```
.shudan-vertex.shudan-marker_point.shudan-sign_-1 .shudan-marker {background: #f00;}
.shudan-vertex.shudan-marker_point.shudan-sign_1 .shudan-marker {background: #f00;}
```

with

```
.shudan-vertex.shudan-marker_point.shudan-sign_-1 .shudan-marker circle {fill: #f00;}
.shudan-vertex.shudan-marker_point.shudan-sign_1 .shudan-marker circle {fill: #f00;}
```

Does that seem reasonable? I can make a PR :)

**Comment by apetresc** [comment]
Hey @upsided, wondering if there's any chance this will be fixed anytime soon. If you're willing, how would you feel about Sabaki upstreaming your themes and maintaining them going forward? Your themes are beautiful and widely used, so it would be great to have a way to keep them up-to-date with Sabaki's DOM changes :) Let me know if that works for you.
